mocked-api/handlers.ts

```diff
@@ -0,0 +1,101 @@
+import { rest } from "msw";
+import { factory, primaryKey } from "@mswjs/data";
+import { BridgeRequest } from "../openapi/generated";
+import { v4 as uuid } from "uuid";
+import { instancesData } from "./data";
+
+// api url
+const apiUrl = `${process.env.BASE_URL ?? ""}${
+  process.env.BASE_PATH ?? "/api/v1"
+}`;
+
+// api response delay in ms
+const apiDelay = 1000;
+
+// set up the model
+const db = factory({
+  bridge: {
+    id: primaryKey(String),
+    name: String,
+    kind: String,
+    href: String,
+    submitted_at: String,
+    published_at: String,
```

Comment:
I left `String` to avoid issues if we reuse the [ts sdk model](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/blob/main/openapi/generated/model/bridge-response.ts#L53) somewhere in the handlers. (they are plain strings there).